### PR TITLE
added simplecov and tests

### PR DESCRIPTION
**Pull request by sarar0**
Please review and let me know if we need more tests. thanks

### CODE REVIEW

**Comment by sarar0** [comment]
[Unit testing on Creation and Reading from database](https://trello.com/c/hpEZI3Ko/29-unit-testing-on-creation-and-reading-from-database)

File: spec/features/02_view_listings_spec.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+feature 'view all listings' do
+    scenario 'user can see the list of all bookmarks' do
+        Listing.create(name: 'test listing three')
+        Listing.create(name: 'test listing four')
+        Listing.create(name: 'test listing five')
+
+        visit '/listings'
+        expect(page).to have_css('li', :text => "test listing three")
+        expect(page).to have_css('li', :text => "test listing four")
+        expect(page).to have_css('li', :text => "test listing five")
```
Comment (Ben230) [inline spec/features/02_view_listings_spec.rb]: is doing this test 3 times necessary? Maybe just once or twice max

Comment (sarar0) [reply spec/features/02_view_listings_spec.rb]: It was to have some sort of a list but we can just have one or two, same result. I have changed and pushed again.

Comment (Ben230) [reply spec/features/02_view_listings_spec.rb]: Just realised this is a feature test, and we have already done this test! please can you delete this file

**Pull request merged**